src/index.html

```diff
@@ -5,9 +5,52 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <a class="card__showcase" href="#">
+        <img
+          class="card__photo"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 product photo"
+        >
+      </a>
+
+      <div class="card__description">
+        <h2 class="card__productName">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__productCod">Product code: 195434</p>
+
+        <div class="card__marks">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__priceTxt">Price:</p>
+          <p class="card__priceNumber">$2,199</p>
+        </div>
+
+        <a
+        href="#"
+        class="button button--txt"
+        data-qa="hover"
+        >
```

Comment:
```suggestion
        <a
          href="#"
          class="button button--txt"
          data-qa="hover"
        >
```
Don't forget about formatting